### ISSUE TITLE
LoRa capability

### ISSUE THREAD

**Issue by montieroland** [ConnectBox/connectbox-pi#336]
Have you considered add LoRa capability so the boxes could form or participate in an ad hoc network?  Maybe talk at a Meshtastic network (https://meshtastic.org/docs/getting-started)

**Comment by kirkdwilson** [comment]
Yes.  WE have test hardware and are considering this verses 802.11ah.   LoRa is a very slow protocol so we are concerned about the value add.  the 802.11ah ;is a fast protocol.  It has more value add.

**Comment by montieroland** [comment]
I can see that. I think LoRa comes in when you want to network ConnectBoxes, maybe an integration with MeshTastic for emergency info or for info across a large campus.

Montie

Montie Roland
***@***.***
Sanford, NC

A candle loses nothing by lighting another candle

Blog and Podcast - www.montie.com
Montie Gear - www.montiegear.com
Simply Us Farm - www.simplyusfarm.com

------- Original Message -------

**Comment by kirkdwilson** [comment]
We are considering 802.11ah as it has higher bandwidth.   In a mesh network this is critical.

**Comment by montieroland** [comment]
If you could add Meshtastic compatibility that could open up a new market that is growing quickly
